### PR TITLE
fix login issue post upgrade

### PR DESCRIPTION
**Pull request by Pearl1594**
## Description
Addresses: https://github.com/apache/cloudstack/issues/4136

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Screenshots (if appropriate):
Upgraded an env from 4.13 to master (4.15) and then installed primate and attempted login. Login happens successfully.

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

**Review by DaanHoogland — APPROVED** [review]
code looks good

**Comment by DaanHoogland** [comment]
@borisstoyanov I think this needs testing explicitely in an upgrade, right?

**Review by borisstoyanov — APPROVED** [review]
LGTM, tested it out with upgrade from 4.13 and it's working fine!

File: plugins/user-authenticators/saml2/src/main/java/org/apache/cloudstack/saml/SAMLUtils.java [hunk 1]
```diff
@@ -280,7 +280,7 @@ public static void setupSamlUserCookies(final LoginCmdResponse loginResponse, fi
             resp.addCookie(new Cookie("timezone", URLEncoder.encode(timezone, HttpUtils.UTF_8)));
         }
         resp.addCookie(new Cookie("userfullname", URLEncoder.encode(loginResponse.getFirstName() + " " + loginResponse.getLastName(), HttpUtils.UTF_8).replace("+", "%20")));
-        resp.addHeader("SET-COOKIE", String.format("%s=%s;HttpOnly;Path=/", ApiConstants.SESSIONKEY, loginResponse.getSessionKey()));
+        resp.addHeader("SET-COOKIE", String.format("%s=%s;HttpOnly", ApiConstants.SESSIONKEY, loginResponse.getSessionKey()));
```
Comment (rhtyd) [inline plugins/user-authenticators/saml2/src/main/java/org/apache/cloudstack/saml/SAMLUtils.java:283]: Thanks, mea culpa.

**Pull request merged**